isofit/radiative_transfer/sRTMnet.py

```diff
@@ -104,12 +104,12 @@ def preSim(self):
         # Emulator Aux
         aux = np.load(config.emulator_aux_file)
 
-        # TODO: Disable when sRTMnet_v100_aux is updated
+        # TODO: Disable when sRTMnet_v120_aux is updated
```

Comment:
Has this been fixed in v120? Can we remove these lines of code? @pgbrodrick

Reply:
Not handled yet.